ocs_ci/deployment/aws.py

```diff
@@ -648,7 +649,7 @@ def remove_rhcos_workers(self):
         for node in rhcos_workers:
             cordon = f"oc adm cordon {node}"
             run_cmd(cordon)
-            drain_nodes(node)
+            drain_nodes([node])
```

Comment:
This fix is part of PR https://github.com/red-hat-storage/ocs-ci/pull/5761 (which should be merged first).